keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -97,6 +97,14 @@ The second iteration, `kubescheduler.config.k8s.io/v1beta2`, includes the follow
     - `RequestedToCapacityRatio` (in favor of `NodeResourcesFit` plugin with a `RequestedToCapacityRatio` scoring strategy)
   - Cleanup of validation hacks.
 
+The third iteration, `kubescheduler.config.k8s.io/v1beta3`, includes the following changes:
+  - Change the weights of the following user configurable plugins.
+    - `InterPodAffinity` to 2
+    - `NodeAffinity` to 2
+    - `TaintToleration` to 3 as the usage of node tainting to group nodes in the cluster is increasing for many user workloads
+
+The main reason for increasing the weights of the above plugins is conflict with other plugins like image locality, utilization.
```

Comment:
The main reason is that some plugins are "default" behavior, whereas these plugins are about user preferences, so should have more relevance.

Reply:
Done